projects/packages/my-jetpack/_inc/components/connected-product-offer/index.jsx

```diff
@@ -67,7 +67,7 @@ const ConnectedProductOffer = ( { slug, onClick, trackButtonClick, ...rest } ) =
 			hasRequiredPlan={ hasRequiredPlan }
 			onAdd={ clickHandler }
 			addProductUrl={ onClick ? undefined : addProductUrl }
-			isLoading={ isFetching }
+			isLoading={ false }
```

Comment:
This component doesn't seem to be used, so I am just removing the isFetching for now. This component will be deleted in a later iteration

Reply:
Maybe "TODO" comment could be useful here? So that it will not get lost anywhere.